apps/address-service/domains/address/schema/AddressSource.test.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * Generated by `createschema address.AddressSource 'source:Text;'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE } = require('@open-condo/keystone/test.utils')
+
+const {
+    expectToThrowAuthenticationErrorToObj,
+    expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+    expectToThrowAccessDeniedErrorToObjects,
+} = require('@open-condo/keystone/test.utils')
+
+const {
+    makeClientWithNewRegisteredAndLoggedInUser,
+    makeClientWithSupportUser,
+} = require('@address-service/domains/user/utils/testSchema')
+
+const {
+    AddressSource,
+    createTestAddressSource,
+    updateTestAddressSource,
+} = require('@address-service/domains/address/utils/testSchema')
+
+describe('AddressSource', () => {
```

Comment:
Could you please write tests on unique constraints

Reply:
Done